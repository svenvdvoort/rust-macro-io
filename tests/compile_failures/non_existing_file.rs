use macro_io::read_file;

fn main() {
    read_file!("tests/testfiles/non-existing-file.txt");
}
