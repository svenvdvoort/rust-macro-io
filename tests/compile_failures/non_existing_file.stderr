error: proc macro panicked
 --> $DIR/non_existing_file.rs:4:5
  |
4 |     read_file!("tests/testfiles/non-existing-file.txt");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: [macro_io::read_file] Could not find file: Os { code: 2, kind: NotFound, message: "No such file or directory" }
